tracer/test/Datadog.Trace.TestHelpers/EnvironmentHelper.cs

```diff
@@ -190,6 +190,9 @@ public static void ClearProfilerEnvironmentVariables()
             string profilerEnabled = AutomaticInstrumentationEnabled ? "1" : "0";
             environmentVariables["DD_DOTNET_TRACER_HOME"] = MonitoringHome;
 
+            // see https://github.com/dotnet/runtime/issues/77973
```

Comment:
```suggestion
            // see https://github.com/dotnet/runtime/issues/77973 and https://github.com/DataDog/dd-trace-dotnet/pull/3579
```

Reply:
Oh yeah, thanks, I meant to find replace all these 😄  I'll link to this PR as that gives the full context 🙂

Reply:
Done in https://github.com/DataDog/dd-trace-dotnet/pull/3579/commits/ae551cdb4af485b9266690296256a3f2c2d39447